partcad/src/partcad/part_factory_step.py

```
@@ -131,7 +137,21 @@
                 time.sleep(0.0001)
                 # TODO(clairbee): remove sleep calls when GIL is fixed in CQ
 
-                shape = cq.importers.importStep(self.path).val().wrapped
+                reader = STEPControl_Reader()
+
+                readStatus = reader.ReadFile(self.path)
+                if readStatus != OCP.IFSelect.IFSelect_RetDone:
+                    raise ValueError("STEP File could not be loaded")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)